src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -14,6 +14,10 @@ $spacing: 20px;
     text-align: center;
   }
 
+  &__wrapper {
+    padding: 0;
+  }
```

Comment:
`.modal-content` is used a lot of times for styling. Needs to reset indents here